blog/_posts/2023-09-28-t31-set-discussion.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: T31 Set Discussion
+#            dd/mm/yyyy
+last_update: 28/09/2023
+excerpt: What does the new tier set do and how does it work?
+author: Elivrio
+toc: true
+---
+
+## T31 effects
+> (2) Set Bonus: {{ site.data.talent.primordial_wave }} also casts an {{ site.data.talent.eb }} at the target at 100% effectiveness, and grants you all three {{ site.data.talent.eb }} bonus effects for 15.0 seconds.
+
+> (4) Set Bonus: {{ site.data.spell.lvb }} burns your target for 15% additional damage over 4 seconds, and after consuming {{ site.data.talent.primordial_wave }}, your next 3 casts of {{ site.data.spell.lvb }} also hit up to 3 additional targets affected by your {{ site.data.spell.fs }} for 100% of normal damage.
+
+First things first. Without being talented into {{ site.data.talent.primordial_wave }} these bonuses do nothing.
+Which means, we're forced into taking this talent for the next tier.
+
+The reworked 4p also puts all of its emphasis on {{ site.data.spell.lvb }}, killing the "weaving" we expected with the first iterations.
+
+<hr>
+
+## 2p
+The additional damage from {{ site.data.talent.eb }} on {{ site.data.talent.primordial_wave }} is a great way to improve the immediate value and feel of the talent.
+While applying {{ site.data.spell.fs }} is a good feature to have, adding the damage makes casting it so much more rewarding.
+
+Furthermore this effect grants all three {{ site.data.talent.eb }} buffs. These present a sizeable chunk of stats.
+
+The additional {{ site.data.talent.eb }} is cast for free. And there exist multiple talents triggered by casting {{ site.data.talent.eb }}.
+Here is a table of them and whether they are triggered by the 2p effect.
+
+Talent | 2p can trigger it
+--- | ---
+{{ site.data.talent.sop }} | ❌ no
+{{ site.data.talent.lr }} | ✅ yes
+{{ site.data.talent.mwf }} | ✅ yes
+{{ site.data.talent.fb }} | ✅ yes
+{{ site.data.talent.wlr }} | ✅ yes
+
+<hr>
+
+## 4p
+
+Any kind of weaving is now gone and you just want to press {{ site.data.spell.lvb }} all day everyday.
+
+Concerning the burn added to {{ site.data.spell.lvb }}, Molten Slag, it functions similarly to Mage's [ignite](https://www.wowhead.com/spell=12654/ignite). It stacks and extends itself if another {{ site.data.spell.lvb }} is used while the target is burning. That's pretty neat and you don't have to play around the debuff duration.
+
+The recent changes to the 4p are good enough to use {{ site.data.spell.lvb }} even when more than 4 targets are present,
```

Comment:
The recent changes to the 4p make {{ site.data.spell.lvb }} strong enough that we want to use it, even when more than 4 targets are present.

Found a comma at the end of a sentence ^^